debian/copyright

```diff
@@ -0,0 +1,52 @@
+Format: http://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+Upstream-Name: Sirupsen/logrus
+Source: https://github.com/Sirupsen/logrus
+
+Files: *
+Copyright: 2014 Simon Eskildsen
+License: MIT
+
+Files: debian/*
+Copyright: 2014 Tianon Gravi <admwiggin@gmail.com>
+License: BSD-2-clause
+
+License: The MIT License (MIT)
```

Comment:
This should just be `License: MIT` (because it's the name created in the `License:` line above, but you should also rename this to be `Expat`, because "The MIT License" doesn't exist and @paultag will eat us if we pretend it does).